api/institutions/views.py

```diff
@@ -402,7 +402,9 @@ class InstitutionSummaryMetrics(JSONAPIBaseView, generics.RetrieveAPIView, Insti
     # overrides RetrieveAPIView
     def get_object(self):
         institution = self.get_institution()
-        return InstitutionProjectCounts.get_latest_institution_project_document(institution)
+        results = InstitutionProjectCounts.get_latest_institution_project_document(institution)
+        results['last_updated'] = UserInstitutionProjectCounts.get_recent_datetime(institution)
```

Comment:
@corbinSanders thinking on this again, it looks like [all metrics should have a `timestamp` field](https://github.com/CenterForOpenScience/django-elasticsearch-metrics/blob/8a02ffc57f57257843834d4f84c41480f4e27fbd/elasticsearch_metrics/metrics.py#L127) -- is it possible to get the timestamp off the "latest" `InstitutionProjectCounts` result that's already being fetched? rather than make additional requests to elastic for `UserInstitutionProjectCounts`

Reply:
👌